schemas/namespace/identity-namespace.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/identity-namespace",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Identity Namespace",
+  "type": "object",
+  "description": "Identity Namespace Schema.",
+  "definitions": {
+    "identity-namespace": {
+      "properties": {
+        "xdm:id": {
+          "title": "Unique Namespace Id",
+          "type": "string",
+          "description": "Unique Identifier for namespace"
+        },
+        "namespaceName": {
+          "title": "Namespace Name",
+          "type": "string",
+          "description": "This is human readable name for the namespace"
+        },
+        "xdm:namespaceCode": {
+          "title": "Namespace Code",
+          "type": "string",
+          "description": "This code is a human readable identifier for the namespace"
+        },
+        "description": {
+          "title": "Namespace Description",
+          "type": "string",
+          "description": "Description for the namespace"
+        },
+        "idType": {
+          "title": "Namespace Id Type",
+          "type": "string",
+          "description": "Identity identifier type for the namespace",
+          "enum": [
+            "EMAIL",
+            "PHONE_NUMBER",
+            "DEVICE",
+            "COOKIE",
+            "CROSS_DEVICE",
+            "NON_PEOPLE"
+          ]
+        },
+        "xdm:createTime": {
+          "title": "Create Time",
+          "type": "string",
+          "description": "The date and UTC Time when the namespace was created"
```

Comment:
also state that it is in ISO 8601 format

Reply:
ack